pegasus/scanner.go

```diff
@@ -100,15 +102,15 @@ func newPegasusScanner(table *pegasusTableConnector, gpid *base.Gpid, partitionH
 	startKey *base.Blob, stopKey *base.Blob) Scanner {
 	gpidSlice := []*base.Gpid{gpid}
 	hashSlice := []uint64{partitionHash}
-	return newPegasusScannerImpl(table, gpidSlice, hashSlice, options, startKey, stopKey)
+	return newPegasusScannerImpl(table, gpidSlice, hashSlice, options, startKey, stopKey, false)
```

Comment:
Why this function doesn't need to check hash?

Reply:
This is used for `hash_scan`, the `hashkey` is specific, no matter is belong to parent or child, its scan result won't have duplicated value, so it needn't check hash.